src/ontology/config/prefixes.csv

```diff
@@ -228,4 +228,6 @@ ICD10WHO,https://icd.who.int/browse10/2019/en#/
 OMIMPS,https://www.omim.org/phenotypicSeries/PS
 OMIM,https://omim.org/entry/
 Orphanet,http://www.orpha.net/ORDO/Orphanet_
-GARD,http://purl.obolibrary.org/obo/GARD_
\ No newline at end of file
+MEDGEN,http://purl.obolibrary.org/obo/MEDGEN_
```

Comment:
## Update: `prefixes.csv` with prefix maps

Reply:
Why remove the GARD prefix?

Reply:
Good to have your eye on this. Looks like I made a copy/paste error. Fixed!